tools/run_all_tests.sh

```diff
@@ -132,16 +122,24 @@ execute_test() {
 
     case $test_scope in
     "all")
+        make all_tests
+        make integration_tests
+        make timing_integrationtests
```

Comment:
does `all_tests` only run all module_tests?

Reply:
No this target runs all module -and integrationtests without timing_tests

Reply:
isn't the `make integration_tests` superfluous? Or should this be `make timing_module_tests`?